### DISCUSSION TITLE
Add a discussion category for slides.com

### DISCUSSION THREAD

**Discussion by ftessier** [hakimel/reveal.js#3513] [Ideas]
I suggest that a discussion category be created in this repo (reveal.js) for discussion and suggestions relating to the slides.com gui (or perhaps a separate discussion-only repo for slides.com?), to take over from UserVoice.

I have been contributing a few suggestions via help.slides.com over the years. However, that UserVoice forum feels clunky and somewhat dated compared to github discussions. Some threads also appear to have been invaded by chatbots (see for example: https://help.slides.com/forums/175819-general/suggestions/3173469-github-integration).

**Comment by t-fritsch** [comment]
As said on https://github.com/hakimel/reveal.js/discussions/3509#discussioncomment-7447107, imho, slides.com content shouldn't live here so that open-source contributors aren't bothered with discussions out of their scope 😅

Maybe another repo may be a good option but I'll let the slides.com team answer that

**Comment by hakimel** [comment]
I agree that this isn't the right place for Slides feedback. I've created a new repo specifically for Slides feedback instead: https://github.com/slides/feedback

@ftessier We're aware of the spam issue on UserVoice. We've been manually deleting a few spam posts per day but it's hard to keep up with. I've been trying to beg the UserVoice team to let us block all content posted with a link but no dice.

We've been considering other options but since all of our help content and support tickets live in UserVoice it's a difficult switch. I'm not sure what the final choice of platform will be for this but I'm happy to use the new GitHub repo for the time being.

**Reply by ftessier** [reply]
Awesome, I will put in my comments in the new repo. I am with you on resolving the bot spam issue and switching your online community forum. We went from G+ to Reddit to Github!
